Ports secret documents from tgstation

nearlyNonexistent opened the pull request:
🆑nearlyNon
rscadd: New items intended for antags to steal; several secret documents that spawn in the SCGR's office, the conference room, and the NTL's office. Gotta collect them all! (Disclaimer: only 3/6 spawn in a game.)
/🆑

Traitor objectives, but not formally. Hints at darker parts of the world.

One out of the 3 SCG options spawns in the conference room, and the liaison and SCGR get their own scripted one.

OhEmGee commented:
these need to be made a bona fide thing

Minijar commented:
I swear these already exist in game for the RD and CO? Unless you're just adding similar things for other heads.

Anton-Kr commented:
We already have sealed envelopes in COs office and RDs iirc.

ghost commented:
And SGR's office.

CakeQ commented:
And NTL's office.

nearlyNonexistent commented:
I didn't see any when poking around at there. Give me a path.

FreePrivateer commented:
Kinda worried about what you mean about 'darker parts of the world'. We don't need to make the game secretly grimdark. Isn't it enough that they would include sensitive information about the ship's mission/capabilities/fleet movements/legal positions?

nearlyNonexistent commented:
@FreePrivateer these were cleared by Eckles and are only as dark as the wiki already states about the Torch project. You can read the descriptions in this PR.

@CakeQ @zaredman @IsdatAfamas Main difference: these allow you to sort of freeform out the exact contents, rather than the implicit meta everyone has with those existing papers. The RD and NTL ones are a different role, ,they direct players to a physical object in the world they can interact with, so they can stay, and the paper itself isn't too important for those. For the SCGR and CO's copies, the issue is that honestly it's too long of a thing and no one has ever used them in round to my knowledge. These can stay too, really. It's not a "remove the current ones and replace with these" situation. One even helps build on the other.

Another part of the purpose of these is to add a pseudoclutter to their desks decoratively. 



Also, even if these were redundant, is it that bad to have more variety?

nearlyNonexistent committed:
Ports secret documents from tgstation

🆑nearlyNon
rscadd: New items intended for antags to steal; several secret documents that spawn in the SCGR's office, the conference room, and the NTL's office. Gotta collect them all! (Disclaimer: only 3/6 spawn in a game.)
/🆑

comma commented:
Yes we have couple lying around but the more the merrier
